tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -169,7 +181,6 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 # Bind PCM with the pipeline
 #
 dnl PCM_PLAYBACK_ADD(name, pcm_id, playback)
-PCM_PLAYBACK_ADD(max373-spk, 0, PIPELINE_PCM_1)
```

Comment:
It's rather odd to remove this PCM playback path?

Reply:
The playback(actually a new echo ref capture also) PCM will be used for smart_amp, and it will be created inside the included file sof-smart-amplifier.m4.

Reply:
ok, so the topology user get the PCM by using SMART_PCM_ID etc. What happens if these are not defined ? The smart amp.m4 file should check that all it's dependency macro are defined and emit error if they are not.

Reply:
yes, it is in here: https://github.com/thesofproject/sof/blob/b51e67f263b1f877869087259e5bf7bdeb328c86/tools/topology/sof-smart-amplifier.m4#L60